chunky/src/java/se/llbit/chunky/renderer/export/Tiff32ExportFormat.java

```diff
@@ -50,8 +52,12 @@ public boolean isTransparencySupported() {
 
   @Override
   public void write(OutputStream out, Scene scene, TaskTracker taskTracker) throws IOException {
+    assert(out instanceof FileOutputStream);
```

Comment:
This is an assumption that violates the method signature. From what I see, a FOS is used to create a file channel to create a buffered output stream on top of it. Why?

Reply:
TIFF format is annoying and uses pointers to addresses which are typically only known after writing parts of the data. I had 2 ways to implement this and chose not to buffer everything in memory before writing it to an unspecified OutputStream. Instead the FileChannel is used to reposition the writer to replace placeholder values for the pointers after writing everything.
Because we currently only use FileOutputStream for PictureExportFormats, it might be better to replace the first argument with a File instead? From that I can create the custom Tiff[...]OutputStream.

Reply:
Allright, but then let's make this a proper runtime exception to yell loudly at whoever dares to mis-use this:


```suggestion
  if (!(out instanceof FileOutputStream)) {
    throw new UnsupportedOperationException("Tiff export is only implemented for FileOutputStream");
  }
```

Reply:
Added a fallback using a temp file

Reply:
That's even better, of course. 👏